hack/install-etcd.sh

```diff
@@ -27,4 +27,4 @@ source "${KUBE_ROOT}/hack/lib/init.sh"
 
 FOUND="$(echo "${PATH}" | sed 's/:/\n/g' | grep -q "^${KUBE_ROOT}/third_party/etcd$" || true)"
 kube::etcd::install
-test -n "${FOUND}" || echo "  PATH=\"\$PATH:${KUBE_ROOT}/third_party/etcd\""
+test -n "${FOUND}" || echo "export PATH=\"\${PATH}:${KUBE_ROOT}/third_party/etcd\""
```

Comment:
this is more useful when pasted by the user, and it also hints that we've exported it if you've sourced the script.